examples/cocoa/CocoaAppDelegate.mm

```diff
@@ -27,218 +28,202 @@
 #import "NSString+stdstring.h"
 
 // These define are used for converting the color of the NSPopUpButton
-#define BLUE    @"Blue"
-#define GREEN   @"Green"
-#define RED     @"Red"
+#define BLUE @"Blue"
+#define GREEN @"Green"
+#define RED @"Red"
 
 #if defined(__APPLE__)
-    #if defined(__clang__)
-        #pragma clang diagnostic ignored "-Wdeprecated-declarations"
-    #elif defined(__GNUC__)
-        #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
-    #endif
+#if defined(__clang__)
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#elif defined(__GNUC__)
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#endif
 #endif
 
 // Our PIMPL
-struct SFMLmainWindow
-{
-    SFMLmainWindow(sf::WindowHandle win) :
-    renderWindow(win),
-    background(sf::Color::Blue)
-    {
-        std::string resPath = [[[NSBundle mainBundle] resourcePath] tostdstring];
-        if (!logo.loadFromFile(resPath + "/logo.png"))
-            NSLog(@"Couldn't load the logo image");
-
-        logo.setSmooth(true);
-
-        sprite.setTexture(logo, true);
-        sf::FloatRect rect = sprite.getLocalBounds();
-        sf::Vector2f size(rect.width, rect.height);
-        sprite.setOrigin(size / 2.f);
-        sprite.scale({0.3f, 0.3f});
-
-        unsigned int ww = renderWindow.getSize().x;
-        unsigned int wh = renderWindow.getSize().y;
-        sprite.setPosition(sf::Vector2f(ww, wh) / 2.f);
-
-        if (!font.loadFromFile(resPath + "/tuffy.ttf"))
-            NSLog(@"Couldn't load the font");
-
-        text.setFillColor(sf::Color::White);
-        text.setFont(font);
-    }
-
-    sf::RenderWindow    renderWindow;
-    sf::Font            font;
-    sf::Text            text;
-    sf::Texture         logo;
-    sf::Sprite          sprite;
-    sf::Color           background;
+struct SFMLmainWindow {
+  SFMLmainWindow(sf::WindowHandle win)
+      : renderWindow(win), background(sf::Color::Blue) {
```

Comment:
This seems to be wrongly formatted. Do Obj-C files take different configs than C++ files?

Reply:
They use the same .clang-format file but I'm guessing our ObjC settings need improvement. Adding `IndentWidth: 4` seems to fix it.

Reply:
Braces placement is still wrong though

Reply:
Right, seems like many options from the C++ section need to get copied into the ObjC section.

Reply:
I think I figured out how to make sure all C++ settings apply to ObjC so that we don't have to repeat ourselves between the two languages.